src/components/TodoList.tsx

```diff
@@ -0,0 +1,21 @@
+import { Todo } from '../types/Todo';
+import { UserInfo } from './UserInfo';
+import { TodoInfo } from './TodoInfo';
+
+type Props = {
+  listOfToDos: Todo[]
+};
+
+export const TodoList: React.FC<Props> = ({ listOfToDos }) => (
+  <ul>
+    {listOfToDos.map(item => (
+      <li key={item.id}>
+        <UserInfo userData={item.user} />
```

Comment:
```suggestion
        {item.user && <UserInfo userData={item.user} />
```
Можна використати conditional rendering і рендерити компонент тільки тоді, коли є поле user